add missing include statement to fix compilation error with Intel compilers:
error: identifier "int32_t" is undefined

author: Kenneth Hoste (HPC-UGent)

--- openmm-7.5.0/openmmapi/include/openmm/internal/vectorize_sse.h.orig	2021-04-21 16:21:13.329507270 +0200
+++ openmm-7.5.0/openmmapi/include/openmm/internal/vectorize_sse.h	2021-04-21 16:22:25.139204132 +0200
@@ -39,6 +39,7 @@
 #endif
 
 #include "hardware.h"
+#include <stdint.h>  // required for int32_t
 
 // This file defines classes and functions to simplify vectorizing code with SSE.
 
